grupo-gomez-lopez/src/components/sharedButton/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+//import sharedButtonCSS from './sharedButtonCSS';
```

Comment:
We shouldn't leave comments in our master branch please delete all comments

I'll be referring to this comment as DC (delete comments)